Diary/Domain.Entities/Domain/BaseTypes/RelatedEntityTypes.cs

```diff
@@ -4,11 +4,11 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Domain.Entities.BaseTypes
+namespace Diary.Core.Domain.BaseTypes
 {
     public enum RelatedEntityTypes
```

Comment:
Enum must have a singular type name - https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Reply:
thanks, i am gonna fix

Reply:
its done